scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -1,69 +1,107 @@
-/* exported populateCytobands */
-function populateCytobands(cytobands){
-  var chromPosPattern = new RegExp("^(?:chr)?([1-9]|1[0-9]|2[0-2]|X|Y|MT)$");
-  var chrom = document.forms["filters_form"].elements["chrom"].value;
-  var chromPos = "";
-  if (typeof document.forms["filters_form"].elements["chrom_pos"] !== "undefined") {
-  	chromPos=document.forms["filters_form"].elements["chrom_pos"].value;
+/* exported sanitizeChromSelOptions() */
+function sanitizeChromSelOptions() {
+	var chrom = [];
+	var chrom_select = document.forms["filters_form"].elements["chrom"];
+  var options = chrom_select && chrom_select.options;
+  var opt;
+
+  // clear other options if All is selected
```

Comment:
Later in the code you have created a function that collects the selected chromosomes (getSelectedChromosomes). perhaps you can reuse it in here? There is some duplicated code that you could remove!

Reply:
Sure! I was just saving some runtime, but it is such a short list it makes no sense.